test/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+add_executable(tester
```

Comment:
tester worked on the first try, hooray! All tests pass

We may want to model this as an actual CMake test, haven't done a whole lot of investigation but it appears we can pass args to an executable so we could just pass in the path to the ion-tests submodule to make it super easy